boards/renesas/rzg3s_smarc/rzg3s_smarc_r9a08g045s33gbg_cm33_defconfig

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2024 EPAM Systems
+# Copyright (c) 2024 Renesas Electronics Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_SYS_CLOCK_EXISTS=y
```

Comment:
should be set at soc level?

Reply:
Thank you.
I moved it to `soc\renesas\rz\rzg3s\Kconfig.defconfig`